README.md

```diff
@@ -256,27 +256,27 @@ A newly spawned agent won't contain any of the docker caches for the first run w
 
 ```yaml
 steps:
-  - label: ":docker: Build an image"
+  - label: ":docker Build an image"
     plugins:
       - docker-compose#v4.16.0:
           build: app
-          push: app:index.docker.io/myorg/myrepo:dev-${BUILDKITE_BUILD_NUMBER}
-          cache-from: app:index.docker.io/myorg/myrepo/myapp:latest
+          push: app:index.docker.io/myorg/myrepo:my-branch
```

Comment:
Should this be `push: app:myregistry:port/myorg/myrepo:my-branch` to match the rest of the example?

Reply:
Once more copied and pasted from the wrong place :)